app/jobs/send_welcome_job.rb

```diff
@@ -0,0 +1,7 @@
+class SendWelcomeJob < ApplicationJob
+  queue_as :mailer
```

Comment:
Would it be worth syncing up the name of the Docker container and this? It's called `bg-emailer` so maybe `bg-mailer`? Or change the queue name here to `mailer`? Just seems like a cheap, but minor, win.

Reply:
Done!